internal/fs/fs.go

```diff
@@ -53,7 +63,8 @@ func HasFilepathPrefix(path, prefix string) bool {
 		return false
 	}
 
-	var d, p string
+	d := vnPath
```

Comment:
It's a combination of many small things. strings.Split later on splits the path. On Windows dirs[0] is the drive letter, on Linux it's empty string. Because of that whoever wrote the function ignored the first element by slicing dirs = (...)[1:] it away. On *nix this is perfect. But on Windows loss of the first element meant that isCaseSensitive later on does not have the full path, i.e. C:\Windows becomes only Windows, and isCaseSensitive becomes unpredictable. I honestly do not know how did the tests previously work. In any case, I had to change the code to remember the volume name and initialize d and p with it. One more thing here. filepath.Join("c:", "Windows") returns "c:Windows". So I had to check if volume name is a drive letter (ending with ':') and only in that situation append \. 

Sorry the explanation is so convoluted, but it's small idiosyncrasies that caused this not to be so trivial.